src/main/java/com/revature/controllers/TestController.java

```diff
@@ -40,7 +46,18 @@
 		return ResponseEntity.ok(new Dog(name, "lab"));
 	}
 	
-
+	@RequestMapping(value="person", method=RequestMethod.GET)
+	public ResponseEntity<?> getAllPersons() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1452)